pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -29,10 +32,24 @@ import (
 )
 
 const (
+	// MaxDisruptedMachinSize is the max size of MachineDisruptionBudgetStatus.DisruptedMachines.
+	// MachineHealthCheck will refuse to delete machine covered by the corresponding MDB
+	// if the size of the map exceeds this value.
+	MaxDisruptedMachinSize = 50
```

Comment:
why 50?

Reply:
I assume if it has more than 50 machines under the `DisruptedMachines` list it says that the cluster has some problem, I  just assume that under the big cluster it can be around 500 nodes, so 50, in this case, looks fine

Reply:
It does not the limit on disruption machines it is a limit on the number of disrupted machines under the `DisruptedMachines map[string]metav1.Time` map, I think the comment describes pretty well the purpose of this map
https://github.com/openshift/machine-api-operator/blob/8b8563a8e9fba4b13babbfd679fc9ab0ed73da0b/pkg/apis/healthchecking/v1alpha1/machinedisruptionbudget_types.go#L36

Reply:
can we please put this in the MDB controller instead of here

Reply:
done

Reply:
done